### PR TITLE
Add support for external databases

### PR DESCRIPTION
**Pull request by shobiz**
This PR includes:
- new environment variable `DB_SERVER` to override the DBSERVER value

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ __You should consider the [use of `--env-file=`](https://docs.docker.com/engine/
 
 * `DB_USER`: username for the database
 * `DB_PASS`: password for the database
+* `DB_SERVER`: DNS for the database location
```
Comment (deitch) [inline README.md]: Can you add here that the default is `db`?

**Comment by deitch** [comment]
This is very nice, thank you! Clean and to the point.

Can we add the default in the `README.md`, and add a test as well? Should be pretty easy to spin up another container with a unique name, and override the backup.

**Comment by shosk33** [comment]
Yeah, that sounds good. I'll add the readme change and test

**Comment by deitch** [comment]
@shobiz why'd you close it? I thought you were going to add the README and test and we were going to merge in?

**Pull request closed without merging**